packages/react-components/src/components/PromoBannerV2/PromoBannerV2.module.scss

```diff
@@ -0,0 +1,71 @@
+$base-class: 'promo-banner-v2';
+
+.#{$base-class} {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: row;
+  border-radius: var(--radius-3);
+  background-color: var(--surface-accent-emphasis-low-info);
+
+  &--small {
+    flex-direction: column;
+
+    .#{$base-class}__content {
+      flex: 0 1 auto;
+      order: 2;
+    }
+
+    .#{$base-class}__additional-content {
+      order: 1;
+    }
+
+    .#{$base-class}__close {
+      padding-top: var(--spacing-3);
+      padding-right: var(--spacing-3);
+    }
+  }
+
+  &__content {
+    display: flex;
+    flex: 0 1 720px;
+    flex-flow: column;
+    justify-content: center;
+    padding: var(--spacing-5);
+    max-width: 720px;
```

Comment:
I think, that we can create a variable (e.g. `$promo-banner-content-max-width`) to use here and a few lines above.

Reply:
+ $promo-banner-height-min (to manage **height** for Horizontal banner)